pkg/actors/core/actor_lock.go

```diff
@@ -11,7 +11,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package actors
+package core
```

Comment:
It's recommended to keep packages as flat as possible unless and only separate if needed. I feel that `core` is not descriptive enough of what the package means. If there's a better package name then lets change to that, if not, I'd rather move that under `actors`.